smartsim/_core/schemas/dragonRequests.py

```diff
@@ -82,4 +82,11 @@ class DragonBootstrapRequest(DragonRequest):
 
 
 @request_registry.register("shutdown")
-class DragonShutdownRequest(DragonRequest): ...
+class DragonShutdownRequest(DragonRequest):
+    # Whether the server should shut down immediately
+    # setting this to False means that the server will
+    # shut down when all jobs are terminated.
+    immediate: bool = True
```

Comment:
Using trailing comments shows up nicely in documentation and in IDE intellisense. Checkout moving them like:

```
  immediate: bool = True
  """Whether the server should shut down..."""
```
and check if your IDE now displays the help text.

See [job.py](https://github.com/CrayLabs/SmartSim/blob/develop/smartsim/_core/control/job.py#L54-L76) for more examples

Reply:
Woah!